src/test/java/redis/clients/jedis/tests/commands/SetCommandsTest.java

```diff
@@ -142,34 +145,46 @@ public void spop() {
   public void spopWithCount() {
     jedis.sadd("foo", "a");
     jedis.sadd("foo", "b");
+    jedis.sadd("foo", "c");
 
-    Set<String> expected = new HashSet<String>();
-    expected.add("a");
-    expected.add("b");
+    Set<String> superSet = new HashSet<String>();
+    superSet.add("c");
+    superSet.add("b");
+    superSet.add("a");
 
     Set<String> members = jedis.spop("foo", 2);
 
     assertEquals(2, members.size());
-    assertEquals(expected, members);
+    assertCollectionContainsAll(superSet, members);
```

Comment:
Why not having a simple comparison and make sure we're getting back the exact two elements we expect?

Reply:
From SPOP, we don't know the exact elements we're going to get if fewer elements are requested.